<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>come√ßa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="velocidade_verde">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,255,0);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="velocidade_vermelho">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,0,0);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="velocidade_amarelo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,0);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="velocidade_azul">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,255);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="velocidade_verde_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,255,0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="velocidade_vermelho_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,0,0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="velocidade_amarelo_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="velocidade_azul_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,255);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>comeca()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocidade_verde</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>atualiza_velocidade_verde()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocidade_vermelho</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>atualiza_velocidade_vermelho()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocidade_amarelo</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>atualiza_velocidade_amarelo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocidade_azul</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>atualiza_velocidade_azul()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocidade_verde_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>diminui_velocidade_verde()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocidade_amarelo_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>diminui_velocidade_amarelo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocidade_azul_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>diminui_velocidade_azul()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocidade_vermelho_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>diminui_velocidade_vermelho()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>muda_posicao()</slot>
  <slot>comeca()</slot>
  <slot>atualiza_velocidade_verde()</slot>
  <slot>atualiza_velocidade_vermelho()</slot>
  <slot>atualiza_velocidade_amarelo()</slot>
  <slot>atualiza_velocidade_azul()</slot>
  <slot>diminui_velocidade_verde()</slot>
  <slot>diminui_velocidade_vermelho()</slot>
  <slot>diminui_velocidade_amarelo()</slot>
  <slot>diminui_velocidade_azul()</slot>
 </slots>
</ui>
